<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPchat</class>
 <widget class="QMainWindow" name="UDPchat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDPchat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="Disconnect_button">
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTextEdit" name="ShowText"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="Send_line"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Local port</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Remote port</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="Port1_line"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Port2_line"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Connect_button">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Send_button">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
